ibis/backends/base_sqlalchemy/mysql/api.py

```diff
@@ -1,6 +1,6 @@
-from ibis.sql.alchemy import to_sqlalchemy
-from ibis.sql.mysql.client import MySQLClient
-from ibis.sql.mysql.compiler import dialect, rewrites  # noqa: F401
+from ibis.backends.base_sqlalchemy import to_sqlalchemy
+from ibis.backends.base_sqlalchemy import MySQLClient
+from ibis.backends.base_sqlalchemy import dialect, rewrites  # noqa: F401
```

Comment:
I think I mentioned in the issue, that better to wait for #2446 to be merged. It's ok to get started, but don't move the mysql backend, since it's move there.

Also, you are changing these imports in a way that doesn't make sense, there are three different modules and you're making them the same?